app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java

```diff
@@ -85,21 +90,21 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         initializeCampusEateryButtons(rootView);
         initializeCollegeTownEateryButtons(rootView);
 
-        mCampusPill.setOnClickListener(new View.OnClickListener() {
+        mCampusPillHolder.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 handleCampusPillPress();
             }
         });
         if (NetworkUtilities.collegetownEateriesLoaded) {
```

Comment:
please use the presenter class instead of accessing the networkutilities variable directly!